documentation/book/con-resource-kafka-connect.adoc

```diff
@@ -0,0 +1,93 @@
+[id='kafka-connect-resource-{context}']
+= Kafka Connect
+
+The operator watches for `KafkaConnect` resource in order to find and get configuration for a Kafka Connect cluster to deploy.
+
+The `KafkaConnectS2I` resource provides configuration for a Kafka Connect cluster deployment using Build and Source2Image features and works only with {OpenShiftName}.
+
+Labels that are applied to the `KafkaConnect` or `KafkaConnectS2I` resources will also be applied to the {ProductPlatformName} resources making up the Kafka Connect cluster. This provides a convenient mechanism for those resource to be labelled in the users preferred way.
+
+The `KafkaConnect` resource supports the following within its `spec`:
+
+.KafkaConnect Parameters
+[cols="30%,50%,20%",options="header"]
+|===
+|Parameter
+|Description
+|Default Value
+
+|`nodes`
```

Comment:
`nodes` -> `replicas`

Reply:
Rechecking if it should be changed from "nodes -> replicas" as the description says "Number of Kafka Connect worker nodes."